app/controllers/home_controller.rb

```diff
@@ -17,5 +18,29 @@ def index
                             .last(num_post - existing_post_num)
                             .reverse
     @posts += most_recent_posts
+
+    @footer_links =
+      [
+        {
+          link: '#link',
```

Comment:
We can add this link :P

Reply:
do we already have a link to the Forums page?